File: exercicios/R/1036/1036.r
input = file('stdin','r')

line = strsplit(readLines(input, n=1), " ")

a = as.double(line[[1]][1])
b = as.double(line[[1]][2])
c = as.double(line[[1]][3])

delta = (b*b) - 4*(a*c)

div = 2 * a

if ((div == 0) | (delta < 0)){
    write(sprintf("Impossivel calcular"),'')
}else{
    r1 = as.double((-b + sqrt(delta)) / (div))

    r2 = as.double((-b - sqrt(delta)) / (div))
    write(sprintf("R1 = %.5f",r1),'')
    write(sprintf("R2 = %.5f",r2),'')
}

File: exercicios/JavaScript/1036/1036.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var line1 = lines.shift().split(' ')

var a = parseFloat(line1[0]);
var b = parseFloat(line1[1]);
var c = parseFloat(line1[2]);

var delta = parseFloat(Math.pow(b,2) - 4*(a*c))

var div = parseFloat(2 * a)

if(div === 0 || div < 0){
    console.log("Impossivel calcular");
}else if(div > 0){
    var r1 = parseFloat((-b + Math.sqrt(delta)) / div)

    var r2 = parseFloat((-b - Math.sqrt(delta)) / div)

    if(r1 != NaN && r2 != NaN){
        console.log("R1 = " + r1.toFixed(5));
        console.log("R2 = " + r2.toFixed(5));
    }
}